buildtest/cli/__init__.py

```diff
@@ -223,12 +223,24 @@ def get_parser():
     )
     parser.add_argument("-r", "--report", help="Specify path to test report file")
 
-    parent_parser = argparse.ArgumentParser(add_help=False)
-    parent_parser.add_argument(
-        "--pager", action="store_true", help="Enable PAGING when viewing result"
-    )
     subparsers = parser.add_subparsers(title="COMMANDS", dest="subcommands", metavar="")
 
+    def get_parent_parser():
+
+        parent_parser = {}
+
+        parent_parser["pager"] = argparse.ArgumentParser(add_help=False)
+        parent_parser["pager"].add_argument(
+            "--pager", action="store_true", help="Enable PAGING when viewing result"
+        )
+
+        parent_parser["file"] = argparse.ArgumentParser(add_help=False)
+        parent_parser["file"].add_argument(
+            "--file", action="store_true", help="Wrtie configuration file to a new file"
```

Comment:
yes the issue is right here i missed this, please remove `action="store_true"` we need argparse to hold value of the file not a `true` or `false`. This should fix the issue